### DISCUSSION TITLE
OMI Champions Meeting 2022-05-06

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#224] [General]
# Agenda
- Take inventory of responsibilities and who is responsible for each
- What gaps do we have with these responsibilities?
- Organizational goals for the next 3 months

**Comment by robertlong** [comment]
# Meeting reminders
- Investigate bots
  - We have Carl bot and Apollo
  - Probably get Carl bot set up to get meeting reminders for all the other regularly occurring meetings
  
# Async communication / work
- Still a problem
- Github is underused
- Processes in place but still seen as confusing

**Reply by Knar33** [reply]
I want to mirror what others said about GitHub: I'd like to learn the open-source process we're following but I find it a bit intimidating. The W3C Code suggests we "Provide alternative ways to contribute".

**Reply by antpb** [reply]
@Knar33 I'm curious what parts that happen in github that you would like to see a proxy way to be involved. I hear a lot of folks say that its hard to get in github, but I have not figured out what parts folks want to do in github that we could make easier elsewhere. Github is a pretty standard way for developers to work so groups like the gltf working group and the scripting group make the most sense in my opinion.

**Reply by Knar33** [reply]
I'm still learning the process so I'm not sure. Not everyone in OMI is a developer though, and even as a developer I've never used GitHub project management tools for open-source contribution. I imagine it's even more intimidating for others who don't use git/GitHub daily.

**Comment by robertlong** [comment]
# Media
  - Evo and LMB currently responsible for leading that effort
  - Evo has other obligations and realistically can only lead for the next couple months
  - LMB generally focused on media strategy
  - We need content
    - Can enlist help from university media programs
  - Goals are to guide the public conversation about an Open Metaverse
  - Can write content about the projects we're actually working on and the progress we're making. How it effects other industries. etc Ex. physics, audio emitter
  - Set regular times to post a regular update
  
 ## Monthly Newsletter
   - Rolled up updates from the last month of OMI meetings
   - Links to posts from the OMI community
   - Summaries of progress in our working groups
   - Process can start with writing a paragraph per working group / project / event in a github issue which we use to compile into the newsletter
   - Long term we should shift to writing these summaries in WordPress
   - @antpb will be the newsletter editor
   - Working group leaders should submit content for their updates (TODO: fill in the blanks with the people responsible)
     - glTF Working Group
     - Scripting Working Group
     - Media Working Group
     - Website Working Group
     - ...
   
 ## Twitter Content
   - Newsletter
   - Events
   - New working group announcements / calls for participation
   - New specs / proposals within groups / request for feedback
   - Who is responsible for our twitter / mastodon account?
   
 ## Blog Content
  - @LightLodges and @antpb to meet and discuss first blog content

**Reply by mrmetaverse** [reply]
https://github.com/omigroup/media/issues/35

**Reply by mrmetaverse** [reply]
Video how-to 

https://github.com/omigroup/media/issues/36

**Comment by robertlong** [comment]
# Organization Administration 
  - Github bot automatically posts to Discord in each corresponding channel
  - @mrmetaverse managing holistic board, message them for any help needed
  
# General Meeting Administration
  - Wednesday general meetings should be led by a rotating group of people
    - @robertlong, @antpb, @LightLodges, @mrmetaverse have volunteered

**Comment by robertlong** [comment]
# Finance
  - Specify process for budgeting
  - Figure out how to get reimbursed / pay for things through Open Collective

**Comment by robertlong** [comment]
# Events
  - Add more community events to our calendar
    - @madjin has been helping update the M3 calendar which we should sync to our calendar somehow
  - knar, Red Eagle,  @mrmetaverse, and @LightLodges have volunteered to lead our next demo night (after June 6th)

**Reply by mrmetaverse** [reply]
I'll volunteer to organize an event like our next demo night. 

Knar volunteered to help as well. 

Evo is willing to be a moderator for the event too (we have had success working together on tools like streamyard together) 

Shadow has expressed interest in leading design events 

Jin has expressed interest in avatar events (like they've been doing with M3!)  

We can continue to partner with other people's events too like we have been.

**Reply by humbletim** [reply]
Re: calendar sync -- not entirely sure if this is an officially support google calendar feature, but it seems possible to overlay multiple embedded calendars using additional `src=` params (`https://calendar.google.com/calendar/u/0/embed?src=src1&src=src2`).

Example: [Combined OMI + M3 Event Calendars](https://calendar.google.com/calendar/u/0/embed?title=Combined+OMI+M3&src=c_0dpugf9f83eq4qekmb6omqbjmg@group.calendar.google.com&src=f9lsrdg825i8ivgifmjts8s4lo@group.calendar.google.com)

**Reply by robertlong** [reply]
Definitely a good stopgap and a good add for our website

**Comment by robertlong** [comment]
# Partnerships / Liaisons
- Discussions about partnerships should happen in public
- We need to be aligned in how we talk to these orgs
- We need to flesh out the interoperator group, code of conduct when acting on behalf of OMI
- We should have people from our group attending other groups meetings as others have done in OMI
- Encourage people to investigate other groups and platforms

**Comment by robertlong** [comment]
# Working Group Administration

  - Create tickets for all actionable items. Working group leaders should be more strict on this.
  - Assign that ticket to to the proper project / column so we can track it (@mrmetaverse manages the holistic OMI board, working groups manage their own board)
  - Weekly agendas / notes need to be taken even if people didn't attend those meetings or nothing on the record was said. That way we can keep track of the health of a group.
  - Website group may want to move to a monthly cadence, most work can be done async
